docs/how-to-guides/data-basics/1-data-basics-setup.md

```diff
@@ -18,7 +18,7 @@ Go ahead and create these files using the following snippets:
 **src/index.js:**
 
 ```js
-import { render } from '@wordpress/element';
+import { render } from 'react';
```

Comment:
Heads-up that similar cases might be found throughout the PR that some APIs are only available in `react-dom`. 💡

Reply:
Good call, I'll check that 👍

Reply:
Should be good now.